### PR TITLE
Community Badges POC

### PR DESCRIPTION
**Pull request by zdrve**
## Description

A quick POC to demonstrate Community Badges 

## Checklist

- [ ] :green_book: all commit messages follow the [conventional commits](https://conventionalcommits.org/) standard
- [ ] :nail_care: SASS files are compiled
- [ ] :arrow_left: changes are compatible with RTL direction
- [ ] :wheelchair: changes are accessible
- [ ] :memo: changes are tested in Chrome, Firefox, Safari, Edge, and IE11
- [ ] :+1: PR is approved by @zendesk/vikings

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -2440,6 +2440,21 @@ ul {
   text-transform: lowercase;
 }
 
+/***** Community Badges *****/
+/* Styles labels used next to the authors of article comments, community posts, and community comments */
+.community-badge {
```
Comment (kaspersorensen) [inline style.css:2445]: Is there a reason that this is duplicated both here and in the `_community-badge.css` file?

Comment (zdrve) [reply style.css:2445]: "style.css" is generated from the styles/*.css files :-)

Comment (kaspersorensen) [reply style.css:2445]: Aaah, got it :D

File: templates/user_profile_page.hbs [hunk 2]
```diff
@@ -31,6 +31,9 @@
         <span class="edit-profile-button">
           {{edit}}
         </span>
+        <span class="assign-badges-button">
+          {{assign_badges}}
```
Comment (kaspersorensen) [inline templates/user_profile_page.hbs:35]: When I try to import this theme, Theme Center tells me that this is invalid.

**Pull request closed without merging**